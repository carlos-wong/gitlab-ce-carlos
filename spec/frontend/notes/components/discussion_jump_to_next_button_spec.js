import { shallowMount } from '@vue/test-utils';
import JumpToNextDiscussionButton from '~/notes/components/discussion_jump_to_next_button.vue';

describe('JumpToNextDiscussionButton', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(JumpToNextDiscussionButton);
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('matches the snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot();
  });
});
