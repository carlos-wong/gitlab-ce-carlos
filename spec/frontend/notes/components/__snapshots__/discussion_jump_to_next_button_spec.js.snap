// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JumpToNextDiscussionButton matches the snapshot 1`] = `
<div
  class="btn-group"
  role="group"
>
  <button
    class="btn btn-default discussion-next-btn"
    title="Jump to next unresolved thread"
  >
    <icon-stub
      name="comment-next"
      size="16"
    />
  </button>
</div>
`;
